com.visualstudio.code-oss.appdata.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+	<id>com.visualstudio.code-oss.desktop</id>
+	<metadata_license>GFDL-1.3</metadata_license>
```

Comment:
Is this using GFDL because it's derived from the upstream app data?

Reply:
No, it is written by me. I did not notice the upstream app data and this may be a better option, thank you!

Reply:
The translations are copied from https://aur.archlinux.org/cgit/aur.git/tree/?h=code